Q/quickfix/build_tarballs.jl

```diff
@@ -0,0 +1,72 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "quickfix"
+version = v"1.16.0"
+
+# Collection of sources required to build CMake
+sources = [
+    GitSource("https://github.com/quickfix/quickfix.git", "92c85ca63fc260d16e24e0ece419ecdec9ffe868"),
+    ArchiveSource("https://github.com/phracker/MacOSX-SDKs/releases/download/10.15/MacOSX10.15.sdk.tar.xz",
+    "2408d07df7f324d3beea818585a6d990ba99587c218a3969f924dfcc4de93b62"),
+    DirectorySource("./bundled")
+]
+
+# needed for libjulia_platforms and julia_versions
+include("../../L/libjulia/common.jl")
+
+#filter julia versions to include only Julia >= 1.8 for LTS
```

Comment:
Why did you choose 1.8 for LTS here? The current LTS release is 1.10, but we can also support older ones if you would like (at least update the comment though).

Reply:
> but we can also support older ones if you would like

We could, but we'd also have loads more builds :smiling_face_with_tear: